templates/web/fixmystreet-uk-councils/about/_sidebar.html

```diff
@@ -0,0 +1,16 @@
+<div class="sticky-sidebar">
+    <aside>
+        <ul class="plain-list">
+            <li>[% INCLUDE link h='/faq' t=loc('Frequently Asked Questions') %]</li>
+            <li>[% INCLUDE link h='/about/posters' t='FixMyStreet Goodies' %]</li>
+            <li>[% INCLUDE link h='/about/house-rules' t='House Rules' %]</li>
```

Comment:
We don't want these two lines in council cobrands. I think it's best to copy `templates/web/base/about/_sidebar.html`, which was the template that was being used, and remove the contact link. It looks like you've copied `templates/web/fixmystreet.com/about/_sidebar.html`, which is just for fixmystreet.com.